### PR TITLE
Feature/pai 42 fe breadcrumbs

### PR DESCRIPTION
**Pull request by SwathiPrasadRaju**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>
Breadcrumb Js Logic
1) Constructed the links based on the url
2) Based on the navigation start level , the breadcrumbs links will populate on the screen
3) we also have hide breadcrumbs where the breadcrumbs will be hidden
4) we also have current page hide option 
Test URLs:

- Before: https://main--pricefx-eds--pricefx.hlx.live/
- After: https://feature-pai-42-fe-breadcrumbs--pricefx-eds--pricefx.hlx.page/index/components/breadcrumbs

### CODE REVIEW

**Review by muneeskumar-rajendran — DISMISSED** [review]

File: blocks/breadcrumb/breadcrumb.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  const pathLink = document.createElement('a');
+  pathLink.href = path.url;
+
+  if (path.name !== 'HomePage') {
+    pathLink.innerText = path.name;
+  } else {
+    pathLink.title = path.label;
+    pathLink.innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" width="24" height="25" viewBox="0 0 24 25" fill="none">
+    <path d="M3 12.5L5 10.5M5 10.5L12 3.5L19 10.5M5 10.5V20.5C5 21.0523 5.44772 21.5 6 21.5H9M19 10.5L21 12.5M19 10.5V20.5C19 21.0523 18.5523 21.5 18 21.5H15M9 21.5C9.55228 21.5 10 21.0523 10 20.5V16.5C10 15.9477 10.4477 15.5 11 15.5H13C13.5523 15.5 14 15.9477 14 16.5V20.5C14 21.0523 14.4477 21.5 15 21.5M9 21.5H15" stroke="#484D56" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+    </svg>`;
```
Comment (Vaitheesaran-nagalingam) [inline blocks/breadcrumb/breadcrumb.js:48]: Please add this SVG icon in Constants.js and use it here.

File: blocks/breadcrumb/breadcrumb.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    if (hideCurrentPage === 'false') {
+      const currentPath = document.createElement('span');
+      currentPath.innerText = document.querySelector('title').innerText;
+      breadcrumbLinks.push(currentPath.outerHTML);
+    }
+
+    breadcrumb.innerHTML =
+      breadcrumbLinks.join(`<span class="breadcrumb-separator"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="25" viewBox="0 0 24 25" fill="none">
+    <path d="M14 5.5L21 12.5M21 12.5L14 19.5M21 12.5L3 12.5" stroke="#484D56" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+    </svg></span>`);
```
Comment (Vaitheesaran-nagalingam) [inline blocks/breadcrumb/breadcrumb.js:81]: Same here as well

**Review by Vaitheesaran-nagalingam — COMMENTED** [review]
Please check my comments

File: blocks/breadcrumb/breadcrumb.js (hunk 1 above)
Comment (SwathiPrasadRaju) [reply blocks/breadcrumb/breadcrumb.js]: Done @Vaitheesaran-nagalingam

File: blocks/breadcrumb/breadcrumb.js (hunk 2 above)
Comment (SwathiPrasadRaju) [reply blocks/breadcrumb/breadcrumb.js]: Done @Vaitheesaran-nagalingam

**Review by Vaitheesaran-nagalingam — APPROVED** [review]

File: blocks/breadcrumb/breadcrumb.js (hunk 1 above)
Comment (Vaitheesaran-nagalingam) [reply blocks/breadcrumb/breadcrumb.js]: Thanks @SwathiPrasadRaju

**Pull request merged**